README.md

```diff
@@ -114,8 +114,8 @@ In summary:
 
 | High Level APIs | Sample use | Arguments         |
 |-----------------|------------|-------------------|
-| QEfficient.cloud.infer           |   [click here](#1-use-qefficientcloudinfer)         |  <li>model_name : $\color{green} {Mandatory}$</li> <li>num_cores : $\color{green} {Mandatory}$</li> <li>device_group : $\color{green} {Mandatory}$</li><li>batch_size : Optional [Default-1]</li> <li>prompt_len : Optional [Default-32]</li> <li>ctx_len : Optional [Default-128]</li><li>mxfp6 : Optional </li> <li>mxint8 : Optional </li><li>hf_token : Optional </li><li>local_model_dir : Optional [Path to custom model weights and config file]</li><li>cache_dir : Optional ["cache_dir" in current working directory]</li><li>**prompt : Optional</li><li>**prompts_txt_file_path : Optional</li><li>verbose : Optional</li>|
-| QEfficient.cloud.execute  |     [click here](#2-use-of-qefficientcloudexcute)       |   <li>model_name : $\color{green} {Mandatory}$</li> <li>device_group : $\color{green} {Mandatory}$</li><li>qpc_path : $\color{green} {Mandatory}$</li><li>prompt : Optional [Default-"My name is"]</li> <li>cache_dir : Optional ["cache_dir" in current working directory]</li><li>hf_token : Optional </li><li>**prompt : Optional</li><li>**prompts_txt_file_path : Optional</li>|
+| QEfficient.cloud.infer           |   [click here](#1-use-qefficientcloudinfer)         |  <li>model_name : $\color{green} {Mandatory}$</li> <li>num_cores : $\color{green} {Mandatory}$</li> <li>device_group : $\color{green} {Mandatory}$</li> <li>**prompt : Optional</li><li>**prompts_txt_file_path : Optional</li> <li>aic_enable_depth_first : Optional </li> <li>mos : Optional [Default=-1]</li> <li>batch_size : Optional [Default=1]</li> <li>prompt_len : Optional [Default=32]</li> <li>ctx_len : Optional [Default=128]</li> <li>generation_len : Optional [Default=None]</li> <li>mxfp6 : Optional </li> <li>mxint8 : Optional </li> <li>local_model_dir : Optional [Path to custom model weights and config file]</li> <li>cache_dir : Optional ["cache_dir" in current working directory]</li> <li>hf_token : Optional </li> <li>verbose : Optional</li>|
```

Comment:
Path to the directory used for saving HuggingFace cache, Default is "efficient-transformers/cache_dir".